README.md

```diff
@@ -86,38 +112,40 @@ use the `artifact-name` parameter:
 ### anchore/sbom-action
 
 The main [SBOM action](action.yml), responsible for generating SBOMs
-and attaching them to your wofklow and releases.
+and uploading them as workflow artifacts and release assets.
 
-| Parameter       | Description                                                                                             | Default                     |
-| --------------- | ------------------------------------------------------------------------------------------------------- | --------------------------- |
-| `path`          | A path on the filesystem to scan. This is mutually exclusive to `image`.                                | \<current directory>        |
-| `image`         | A container image to scan. This is mutually exclusive to `path`.                                        |
-| `artifact-name` | The name to use for the generated SBOM artifact. See: [Naming the SBOM output](#naming-the-sbom-output) | `sbom-<job>-<step-id>.spdx` |
-| `format`        | The SBOM format to export. One of: `spdx`, `spdx-json`, `cyclonedx`                                     | `spdx-json`                 |
+| Parameter           | Description                                                                                                                                  | Default                     |
+| ------------------- | -------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------- |
+| `path`              | A path on the filesystem to scan. This is mutually exclusive to `image`.                                                                     | \<current directory>        |
+| `image`             | A container image to scan. This is mutually exclusive to `path`. See [Scan a container image](#scan-a-container-image) for more information. |
+| `registry-username` | The registry username to use when connecting to an external registry                                                                         |
+| `registry-password` | The registry password to use when connecting to an external registry                                                                         |
```

Comment:
nit: I might say "authenticating" in place of "connecting" in these parameter descriptions. Just a soft opinion though